src/ext/resources/gtfsgraphqlapi/schema.graphqls

```diff
@@ -3221,6 +3553,19 @@ enum RealtimeState {
     MODIFIED
 }
 
+scalar Reluctance
+
+input InputRentalPreferences {
+    allowKeepingVehicleAtDestination: Boolean
+    keepingVehicleAtDestinationCost: Cost
+    allowedNetworks: [String!]
+    bannedNetworks: [String!]
+    pickupTime: Duration
+    pickupCost: Cost
+    dropOffTime: Duration
+    dropOffCost: Cost
```

Comment:
I wonder if we should expose these or not. Same for the similar parking parameters.